### PR TITLE
Add a parse_membership_data command

### PR DESCRIPTION
**Pull request by tmcadam**
Add a command to to parse the membership dump files created by dumpstats and upload them to the nci_resources_16 database in Nectar. Linked to https://github.com/ACCESS-NRI/dumpstats/pull/45

### CODE REVIEW

File: ncigrafana/parse_membership.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+        # parse project_id and members from file
+        data_line_parts = data_line.strip().split(":")
+        project = data_line_parts[0]
+        members_csv = data_line_parts[3]
+        members_list = members_csv.split(",")
+
+        # remove service users
+        members_list = [ m for m in members_list if not m.endswith("_thredds") ]
+        members_list = [ m for m in members_list if not m.endswith("_ci") ]
```
Comment (CodeGat) [inline ncigrafana/parse_membership.py:63]: Note that other service users might not follow the pattern `*_ci`

**Review by CodeGat — APPROVED** [review]

File: ncigrafana/parse_membership.py (hunk 1 above)
Comment (aidanheerdegen) [reply ncigrafana/parse_membership.py:63]: `ki32` is a good project to check out the possible spectrum of service users, and lordy!, there is no pattern at all
```
accesstester 
hc46_gitlab 
ki32_nfs
tm70_ci
wp25_ci 
yj2_repoweb
```

I looked in the LDAP record, and the only thing I could find that was different about the service users is they don't have either the `mail:` field or `labeledURI` field.

My suggestion is to not exclude service users at this point, but add a column in the users table to indicate service users and just manually enter the ones we know, and filter them out of reports as required.

We can try automating that process by checking if the above fields exist, but that is problematic if they decide to change how the LDAP records are returned.

File: ncigrafana/parse_membership.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    project = None
+    date = None
+    system = 'gadi'
+
+    with open(filename) as f:
+
+        print("Parsing {file}".format(file=filename))
+
+        # read 3 lines in membership dump file
```
Comment (aidanheerdegen) [inline ncigrafana/parse_membership.py]:
```suggestion
        # read first line in membership dump file
```

**Review by aidanheerdegen — COMMENTED** [review]
One tiny change to a commented line. Looks good pending changes to service users (or not) from Tommy's review.

File: ncigrafana/parse_membership.py (hunk 1 above)
Comment (tmcadam) [reply ncigrafana/parse_membership.py:63]: I will keep an out for others

**Pull request merged**